datarepo-clienttests/src/main/java/scripts/testscripts/DRSLookup.java

```diff
@@ -43,6 +44,13 @@ public DRSLookup() {
   private static List<BlobId> scratchFiles = new ArrayList<>();
   private String dirObjectId;
 
+  private int K = 1;
+
+  public void setParameters(List<String> parameters) {
+      K = Integer.parseInt(parameters.get(0));
+      logger.debug("Repeated DRS Lookups (default is 1): {}", K);
```

Comment:
Is it 1?  Where does that get set?

Reply:
On line 47, yes?

Reply:
derp.  Right.  Though you should still have a more meaningful name for the variable